cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/DeployProgress/DeployTimeline/DeployTimeline.scss

```diff
@@ -0,0 +1,122 @@
+// Clarity Core does not include the 'timeline' component as of 05/22
+// This copies majority of the clarity timeline css styles to avoid importing the clr-ui library
+
+.cds-timeline.cds-timeline-vertical {
+    min-width: 9rem;
+}
+
+.cds-timeline {
+    display: flex;
+    margin: 0;
+    padding: 0 0.6rem;
+}
+
+.cds-timeline-step {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    min-width: 8.75rem;
+    margin-left: 0.6rem;
+
+    cds-icon,
+    cds-icon {
+        height: 1.8rem;
+        width: 1.8rem;
+        min-height: 1.8rem;
+        min-width: 1.8rem;
+    }
+
+    cds-icon[shape='circle'],
+    cds-icon[shape='circle'] {
+        // TODO: enable light mode css
+        // color: #737373; <- light mode
+    }
+
+    cds-icon[shape='dot-circle'],
+    cds-icon[shape='dot-circle'] {
+        color: var(--cds-alias-status-info);
+    }
+
+    cds-icon[shape='success-standard'],
+    cds-icon[shape='success-standard'] {
+        color: var(--cds-alias-status-success);
+    }
+
+    cds-icon[shape='error-standard'],
+    cds-icon[shape='error-standard'] {
+        color: var(--cds-alias-status-danger);
+    }
+
+    // style for the connecting line between icons
+    &:not(:last-of-type) .cds-timeline-step-body::before {
+        content: '';
+        background: var(--cds-global-color-warm-gray-200);
+        height: 0.1rem;
+        width: calc(100% - 0.9rem - 0.1rem);
+        transform: translate(1.7rem,-0.95rem);
+    }
+}
+
+.cds-timeline-step-title {
+    font-size: .65rem;
```

Comment:
There are some constants for font-size, font-weight in clarity.

Reply:
Updated, thanks for the attention to detail on that